BrenKt-SK-997-TRANS-WEB-hosting-change_offer (#3985)
ovh/docs: pages/web/hosting/how_to_upgrade_web_hosting_offer/guide.en-gb.md

Then select your new subscription and its duration. Confirm the corresponding contracts, then click `Send`{.action}.

### Checking that your hosting plan is compatible with a lower plan <a name="checks"></a>

> [!warning]
>

> > You can only change your subscription to a lower range plan if it is the **lower** range plan.

Reviewer: ```suggestion
> You can only change your subscription to a lower range plan if it is the immediate **lower** range plan.
```

![mxplan](images/mxplan.png){.thumbnail}

#### 6 - Mailing lists

The [Mailing lists](https://docs.ovh.com/gb/en/emails/guide-dutilisation-mailing-list/) feature is optional on [Personal](https://www.ovhcloud.com/en-gb/web-hosting/personal-offer/) and [Kimsufi Web](https://www.ovhcloud.com/en-gb/web-hosting/old-web-hosting-offers/) hosting plans.

To set up your hosting plan on a [Personal](https://www.ovhcloud.com/en-gb/web-hosting/personal-offer/) solution, you will need to delete the mailing lists first, or order an email solution with this feature (**MX Plan 100** or **MX Plan Full**) from your [OVHcloud Control Panel](https://www.ovh.com/auth/?action=gotomanager&from=https://www.ovh.co.uk/&ovhSubsidiary=GB).

In the `Emails`{.action} section of your Control Panel, select the solution concerned, then click on `...`{.action} to the right of `Solution`{.action}. Next, click `Change solution`{.action}.

#### Completion

Once you have checked these 6 elements, you can [change your plan](#modify).

### Special cases

#### You have a Start 10M solution <a name="start10m"></a>

> If you change your [Start10M](https://docs.ovh.com/gb/en/hosting/activate-start10m/) solution, you will only be offered the [Personal Hosting](https://www.ovhcloud.com/en-gb/web-hosting/personal-offer/) plan. However, once you have switched to the Personal hosting plan, you can migrate it to all of our [web hosting plans](https://www.ovhcloud.com/en-gb/web-hosting/).

Reviewer: ```suggestion
If you change your [Start10M](https://docs.ovh.com/gb/en/hosting/activate-start10m/) solution, you will only be offered the [Personal Hosting plan](https://www.ovhcloud.com/en-gb/web-hosting/personal-offer/). However, once you have switched to the Personal hosting plan, you can migrate it to all of our [web hosting plans](https://www.ovhcloud.com/en-gb/web-hosting/).
```

Please follow [these instructions](#modify) to change your plan.

#### Boost your Performance web hosting plan temporarily <a name="boost"></a>

With the [Boost option](https://www.ovhcloud.com/en-gb/web-hosting/options/boost/), available on our *Performance* plans, you can temporarily increase your hosting system’s CPU and RAM resources to absorb seasonal traffic spikes. If this increase is prolonged over time, you can also [switch to the higher-level Performance plan](#modify) to have these resources permanently.

> [!warning]
>

> > When you choose to enable the Boost option, the option remains active and charged **until you disable it**.

Reviewer: ```suggestion
> When you choose to enable the Boost option, the option remains active and is billed **until you disable it**.
```

If the **Boost** option suits your needs, below are the instructions for **enabling** or **disabling** this option on your web hosting plan.

> [!tabs]
> **Enable the Boost option**
>>
>> In the `General informations` section of your Web Hosting plan, click on the `...`{.action} button to the right of `Boost`, then `Boost my plan`{.action}.<br><br>
>> ![boost](images/enable_boost.png){.thumbnail}<br>
>>
> **Disable the Boost option**
>>
>> In the `More` tab of your web hosting plan, click `Boost my plan`{.action}.<br>
>> The Boost option usage table will appear, click `Disable boost plan`{.action}.<br><br>
>> ![boost](images/disable_boost.png){.thumbnail}<br>

#### Billing in case of a change of offer <a name="billing"></a>

When you upgrade your original plan to a higher plan, a *prorata temporis* calculation will be applied until the next renewal date for your original plan.
This calculation corresponds to the price difference between your initial solution and your new solution.

> **Example:**<br>
>
> You subscribed to a [Personal Hosting](https://www.ovhcloud.com/en-gb/web-hosting/personal-offer/) subscription on January 1, 2022.
>

> > On October 31, 2022, you will upgrade from this **Personal** plan to a subscription on the plan >

Reviewer: The end of this sentence is missing. Please check also the other translations for the same bug.
